src/components/ui/SportsWinnersContainer.tsx

```diff
@@ -0,0 +1,131 @@
+import { cva, type VariantProps } from 'class-variance-authority';
+import { FC } from 'react';
+
+//Border & Text Colors
+const borderTextStyles = cva(['font-chakra-petch'], {
+  variants: {
+    color: {
+      blueBlue: 'border-blue text-blue',
+      blueWhite: 'border-blue text-white',
+      greenBlue: 'border-green text-blue',
+      greenPink: 'border-green text-pink',
+      pinkGreen: 'border-pink text-green',
+      pinkWhite: 'border-pink text-white',
+      whiteBlue: 'border-white text-blue',
+      whitePink: 'border-white text-pink',
+    },
+  },
+  defaultVariants: {
+    color: 'blueBlue',
+  },
+});
+
+//Title Bakcground & Title Color
+const titleStyles = cva('font-chakra-petch', {
+  variants: {
+    titleColor: {
+      blueGreen: 'bg-blue text-green',
+      pinkGreen: 'bg-pink text-green',
+      blueWhite: 'bg-blue text-white',
+      pinkWhite: 'bg-pink text-green',
+      greenBlue: 'bg-green text-blue',
+      whiteBlue: 'bg-white text-blue',
+      greenPink: 'bg-green text-pink',
+      whitePink: 'bg-white text-pink',
+    },
+  },
+  defaultVariants: {
+    titleColor: 'greenBlue',
+  },
+});
+
+interface Winners {
+  id: number;
+  game_title: string;
+  first_place_name: string;
+  second_place_name: string;
+  third_place_name: string;
+  color:
+    | string
+    | 'blueBlue'
+    | 'blueWhite'
+    | 'greenBlue'
+    | 'greenPink'
+    | 'pinkGreen'
+    | 'pinkWhite'
+    | 'whiteBlue'
+    | 'whitePink';
+  titleColor:
+    | string
+    | 'greenBlue'
+    | 'greenPink'
+    | 'whiteBlue'
+    | 'whitePink'
+    | 'blueGreen'
+    | 'blueWhite'
+    | 'pinkGreen'
+    | 'pinkWhite';
```

Comment:
instead of doing this you can set color and titleColor by using Variant Props

// Define types for color and titleColor props using VariantProps
type BorderTextColorVariants = VariantProps<typeof borderTextStyles>;
type TitleColorVariants = VariantProps<typeof titleStyles>;

// Define Winners interface using the types extracted from VariantProps
interface Winners {
  id: number;
  game_title: string;
  first_place_name: string;
  second_place_name: string;
  third_place_name: string;
  color: BorderTextColorVariants['color'];
  titleColor: TitleColorVariants['titleColor'];
}